### PR TITLE
Users/stevenh/zlux 500

### PR DESCRIPTION
**Pull request by stevenhorsman**
## PR Type
- [ ] Bug fix
- [X] Feature
- [ ] Other (Please indicate)

Add keyring support back in and update build: https://github.com/zowe/zlux/issues/500

### CODE REVIEW

**Comment by stevenhorsman** [comment]
Integrated in zip branch build: https://wash.zowe.org:8443/job/zowe-install-packaging/job/users%252Fstevenh%252Fexplorer-keyring-test/
Convenience test: https://wash.zowe.org:8443/job/zowe-install-test/job/users%252Fstevenh%252Fexplorer-keyring-test/2/
SMP/E test: https://wash.zowe.org:8443/job/zowe-install-test/job/users%252Fstevenh%252Fexplorer-keyring-test/3/

File: package-lock.json [hunk 1]
```diff
@@ -6436,6 +6436,15 @@
         "path-exists": "^3.0.0"
       }
     },
+    "keyring_js": {
```
Comment (NakulManchanda) [inline package-lock.json:6439]: we don't have keyring_js in package.json (as direct dependency) of explorer jes, i believe this should not be here

Comment (NakulManchanda) [reply package-lock.json:6439]: this **commit hash above not required 1dbffa2**
is not required

**Comment by NakulManchanda** [comment]
basically we should rollback package.json and then ok to merge, thanks for testing and fixing pipeline

**Pull request merged**